PowerPointLabs/Test/FunctionalTest/ConvertToPictureTest.cs

```diff
@@ -50,5 +51,28 @@ private static void ConvertSingleObjToPicture()
             PpOperations.SelectShape("text 3")[1].Delete();
             SlideUtil.IsSameLooking(expSlide, actualSlide);
         }
+
+        private static void CheckIfClipboardIsRestored()
```

Comment:
Nice test function. Can this be reused for pastelab, etc tests too?

Reply:
Yes it can be reused! However for PasteLab, only ReplaceWithClipboard and PasteIntoGroup will make use of CorruptionCorrection (which in turn uses RestoreClipboardAfterAction). I'll try to add it to PasteLab tests.